js/client-teambuilder.js

```diff
@@ -2732,23 +2776,6 @@
 			}
 			this.save();
 		},
-		unChooseMove: function (moveName) {
```

Comment:
This method was just cleanup logic to undo any IV changes we may have made when selecting a move, but now we don't make any such changes in the first place so there's no point in keeping it around